projects/storefrontlib/src/cms-components/configurator/commons/config-attribute-types/config-attribute-numeric-input-field/config-attribute-numeric-input-field.service.ts

```diff
@@ -135,19 +143,25 @@ export class ConfigAttributeNumericInputFieldService {
           locale,
           NumberSymbol.Decimal
         );
-
+        const expressionPrefix = negativeAllowed ? '^-?' : '^';
         const expressionOnlyNumericalInput: RegExp = new RegExp(
-          '^-?[0123456789' + groupingSeparator + decimalSeparator + ']+$'
+          expressionPrefix +
+            '[0123456789' +
+            groupingSeparator +
+            decimalSeparator +
+            ']*$'
         );
+
         if (!expressionOnlyNumericalInput.test(input)) {
           return this.createValidationError(true);
         }
+        console.log('CHHI accepted');
```

Comment:
Christoph, could you remove a console log? Thanks.

Reply:
Done!